ui/app/AppLayouts/Wallet/controls/ManageTokensGroupDelegate.qml

```diff
@@ -13,18 +13,18 @@ DropArea {
     id: root
     objectName: "manageTokensGroupDelegate-%1".arg(index)
 
-    // expected roles: communityId, communityName, communityImage
+    // expected roles: communityId, communityName, communityImage, collectionUid, collectionName, imageUrl // FIXME unify group image
 
     property int visualIndex: index
     property var controller
-    property bool communityGroupsExpanded
     property var dragParent
     property alias dragEnabled: groupedCommunityTokenDelegate.dragEnabled
-    property bool isCollectible
+    property bool isCollectible: isCollection
+    property bool isCollection
     property bool isHidden // inside the "Hidden" section
 
-    readonly property string communityId: model.communityId
-    readonly property int childCount: model.enabledNetworkBalance // NB using "balance" as "count" in m_communityTokenGroupsModel
+    readonly property string groupId: isCollection ? model.collectionUid : model.communityId
```

Comment:
Could this be assigned from outside? Following the same thoughts than before, should the component be agnostic of the model type?